rpcs3/Emu/Cell/lv2/sys_net/lv2_socket.cpp

```diff
@@ -81,6 +93,15 @@ s32 lv2_socket::clear_queue(ppu_thread* ppu)
 		{
 			it = queue.erase(it);
 			cleared++;
+
+			// Makes sure network_context thread can go back to sleep if there is no active polling
+			if (type == SYS_NET_SOCK_STREAM || type == SYS_NET_SOCK_DGRAM)
+			{
+				auto& nc = g_fxo->get<network_context>();
+				ensure(nc.num_polls != 0);
+				nc.num_polls--;
```

Comment:
ugh, when you do it like that you miss the atomicity check which guarentees it won't underflow. But it's not a blocker, if there would be an isdue, this verification would fail 99% of the time it should.

Reply:
Changed it to a fetch_sub to ensure a 0 is always caught.